README.md

```diff
@@ -32,6 +32,10 @@ behalf using the [GitHub
 OAuth](https://developer.github.com/v3/oauth/#web-application-flow) flow.
 Alternatively, users can comment on the GitHub issue directly.
 
+## Build requirements
+
+Python 3.10 is required to build the web pages.
```

Comment:
```suggestion
Python 3.10 is required to build the web pages. It might be possible to use a later version once an yapsy
update has been released (see https://github.com/tibonihoo/yapsy/issues/19).
```

Reply:
I suggest instead to create an issue on the tracker for upgrading to Nikola 8.3.0.

Reply:
Whoops. My comment should have gone here. This is a good idea. Thanks for opening the issue.